client/src/components/InputTextField/InputTextField.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from "react";
+import "./InputTextField.css";
+import searchIcon from "../../../../assets/images/global-nav_search_icon.png";
+/**
+ * Created by Tyler
+ *
+ * Props will define the type of input field
+ * Input fields include email, password, name, confirm password, city of residence, what is your age?, what is your dream destination, First name,   last name, search my directory, search trips.
+ *
+ * I think we'll need to implement a function on these to retrieve the data typed in when pressing a submit button. I wasn't sure how to do this with just the standalone components in storybook.
+ */
+
+// Class Based React Component
+class InputTextField extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "input-text-field-container",
+      classListSmall: "input-text-field-container-small",
+      classListSearch: "input-text-field-container search",
+      email: "",
+      name: ""
+    };
+  }
```

Comment:
I don't think this component will need any state - it should take a `onChange` method as a `prop`, but shouldn't manage it's own state. The goal here (of all atomic components) is to just render the piece of the UI. The component rendering it will be responsible for state. Does that make sense?

Reply:
Understood, working on changes now. Thanks!